devops/gce-nested/gce-runner.sh

```diff
@@ -51,10 +51,10 @@ function copy_securedrop_repo() {
 
 # Main logic
 copy_securedrop_repo
+ssh_gce "sudo apt install python3-dev build-essential"
```

Comment:
During standup today, @emkll noted that this line is causing CI to fail. The problem is two-fold:

* the dependencies listed here should be baked into the base box used for CI (to save time on CI runs)
* the `sdci` user running these commands lacks sudo rights, so ad-hoc apt install tasks will always fail

Both should be resolved. Will handle as part of review. The changes to the base box won't be reflected in the diff for this PR, but once confirmed working, we can drop the `sudo apt install` task here prior to merge.